Add missing template list expanded section details

marshmalien opened the pull request:
##### SUMMARY
Issue: #7662

* Add the following detail items:
  * Description
  * Organization and link to organization details
  * Credentials
* Move Credential and Label details to the bottom of the expanded section and make them stretch the entire width of the row. 

<img width="1100" alt="Screen Shot 2021-04-07 at 1 58 51 PM" src="https://user-images.githubusercontent.com/15881645/114090063-f0eb7580-9884-11eb-94d6-9eb32e5c932c.png">

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
 - UI

marshmalien committed:
Add remaining expanded template list item details

nixocio reviewed (approved):
@marshmalien 

I triggered the e2e tests. 

I noticed that we are displaying empty `activity`. In general we do not display empty fields, but for the activity one make sense. Right?
Also the EE field would be a good one to be displayed as part of the expand section details. It was introduced later on.

marshmalien committed:
Add execution environment detail to template expanded row

marshmalien committed:
Fix bug where falsy details return 0

marshmalien commented:
@nixocio We probably didn't show an empty "Activity" detail intentionally, so now we should check for recent_jobs before rendering "Activity". I also added Execution Environment detail to the expanded content and fixed a bug where short-circuit falsy values rendered 0 instead of nothing. 
<img width="1595" alt="Screen Shot 2021-04-09 at 3 36 38 PM" src="https://user-images.githubusercontent.com/15881645/114232717-55243d00-994a-11eb-98ab-33fcd6d1ec2d.png">
